src/templates/admin/core/article.html

```diff
@@ -22,12 +22,13 @@ <h2>{{ article.title }}</h2>
         </div>
     </div>
 
+    {% if not journal_settings.general.hide_review_data_pre_release or article.peer_reviews_for_author_consumption %}
     <div class="box">
         <div class="title-area">
             <h2>Reviews</h2>
         </div>
         <div class="box">
-            {% if journal_settings.general.enable_peer_review_data_block %}
+            {% if journal_settings.general.enable_expanded_review_details %}
```

Comment:
What's the difference between these settings? Have we documented them? I think we need an explainer on how to use these settings in combination, since we now have so many:

- `hide_review_data_pre_release`
- `enable_peer_review_data_block`
- `enable_expanded_review_details`